tests/CouchDB.Driver.UnitTests/UnsupportedMethods_Tests.cs

```diff
@@ -0,0 +1,221 @@
+            }
+        }
+        [Fact]
+        public async Task Any()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.AsQueryable().Any();
+                Assert.True(result);
+            }
+        }
+        [Fact]
+        public async Task Any_Selector()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.AsQueryable().Any(r => r.Name == _mainRebel.Name);
+                Assert.True(result);
+            }
+        }
+
+        [Fact]
+        public async Task Avg_Expr()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.AsQueryable().Average(r => r.Age);
+                Assert.Equal(19, result);
+            }
+        }
+
+        [Fact]
+        public async Task Count()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.AsQueryable().Count(r => r.Age == 19);
+                Assert.Equal(1, result);
+            }
+        }
+
+        [Fact]
+        public async Task DefaultIfEmpty()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.Where(c => c.Name == "Luce").DefaultIfEmpty(_mainRebel);
+                Assert.Equal(_mainRebel, result.First());
+            }
+        }
+
+        [Fact]
+        public async Task ElementAt()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.AsQueryable().ElementAt(0);
+                Assert.Equal(_mainRebel, result);
+            }
+        }
+        [Fact]
+        public async Task ElementAt_OrDefault()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.AsQueryable().ElementAtOrDefault(2);
+                Assert.Null(result);
+            }
+        }
+
+        [Fact]
+        public async Task First()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.AsQueryable().First();
+                Assert.Equal(_mainRebel, result);
+            }
+        }
+        [Fact]
+        public async Task First_Expr()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                httpTest.RespondWithJson(_response);
+                var result = _rebels.AsQueryable().First(c => c.Name == _mainRebel.Name);
```

Comment:
Could this be run in CouchDb by interpreting as `_rebels.Where(c => c.Name == _mainRebel.Name).Limit(1)` and grabbing the 1st item?